marketplace/adoptium-marketplace-client/src/test/java/net/adoptium/marketplace/client/RepoTest.java

```diff
@@ -0,0 +1,34 @@
+package net.adoptium.marketplace.client;
+
+import net.adoptium.marketplace.client.signature.SignatureType;
+import net.adoptium.marketplace.schema.ReleaseList;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class RepoTest {
+    @EnabledIfEnvironmentVariable(named = "VALIDATE_REPO", matches = ".*")
+    @Test
+    public void validateRepo() throws IOException {
+        String publicKey = Files.readString(Path.of("../exampleRepositories/keys/public.pem"));
+
+        String repoUrl = "http://localhost:8000/";
+
+        try {
+            MarketplaceClient client = MarketplaceClient.build(repoUrl, SignatureType.BASE64_ENCODED, publicKey);
+            ReleaseList releaseList = client.readRepositoryData();
+
+            System.out.println("Found: " + releaseList.getReleases().size() + " releases");
```

Comment:
system out and system err?  Not a logging framework?

Reply:
Going for a compact example with as few dependencies as possible as this is to some extent code I expect vendors to copy and paste to validate their repos